docs/config.md

```diff
@@ -28,15 +28,18 @@ The following fields are supported:
     * Typically named `vmlinuz` or `bzImage`
     * If a relative path is provided, it will be interpreted as relative to
       `vmtest.toml`
+    * Should have `CONFIG_9P_FS` and `CONFIG_NET_9P_VIRTIO` enabled if `rootfs`
+      is in use.
```

Comment:
How about linking to README instead? https://github.com/danobi/vmtest?tab=readme-ov-file#dependencies 

We're working on moving to virtiofs instead of 9pfs. So would be good to keep all the references centralized.

Reply:
Oh nice, I forgot about that, will do.